servicetalk-encoding-netty/src/test/java/io/servicetalk/encoding/netty/NettyChannelContentCodecTest.java

```diff
@@ -76,41 +76,31 @@ public NettyChannelContentCodecTest(final ContentCodec codec) {
 
     @Test
     public void testEncode() {
-        testEncode(DEFAULT_ALLOCATOR, 0);
+        testEncode(DEFAULT_ALLOCATOR);
     }
 
     @Test
-    public void testEncodeWithOffset() {
-        testEncode(DEFAULT_ALLOCATOR, 10);
+    public void testEncodeWithReadOnlyBuffer() {
+        testEncode(DEFAULT_RO_ALLOCATOR);
     }
 
     @Test(expected = CodecEncodingException.class)
     public void testEncodeWithOffsetAndZeroLength() {
-        testEncode(DEFAULT_ALLOCATOR, 10, 0);
-    }
-
-    @Test(expected = CodecEncodingException.class)
-    public void testEncodeWithOffsetAndOverflowLength() {
-        testEncode(DEFAULT_ALLOCATOR, 1023, 0);
```

Comment:
Not necessary for this PR (I do not expect anyone uses codec api directly), but for future cases when you deprecate API, consider removing tests at the time you remove the method. It helps to make sure the old API still works as expected before we remove it.